docs/source/samples/SpatialDetection/spatial_tiny_yolo.rst

```diff
@@ -1,7 +1,7 @@
 RGB & TinyYolo with spatial data
 ================================
 
-This example shows how to run TinyYoloV3 and v4 on the RGB input frame, and how to display both the RGB
+This example shows how to run Yolo on the RGB input frame, and how to display both the RGB
```

Comment:
Technically tiny yolo

Reply:
True, but some users have a feeling that only TinyYolo V3/V4 are supported, and that's not really the case here. You can change the model easily